### PR TITLE
feat: Expose CoreAppPlane JobRunner codebuild IAM role

### PR DESCRIPTION
**Pull request by ujwalbukka**
### Reason for this change

We need this because if codebuild project is provisioning a resource for an AWS service. The resource policy of a AWS service will expect the codebuild project role has permissions to do the needful. 
For example, the if codebuild project is creating an index in OpenSearch serverless then data access policy of OpenSearch serverless expects the codebuild project role is mentioned as principal which has access to create index.

### Description of changes

Created a public readonly map which has jobname and jobRunnerRoleArn

### Description of how you validated changes

added unit test

### Checklist

- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/awslabs/sbt-aws/blob/main/CONTRIBUTING.md)

---

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.*

### CODE REVIEW

File: test/core-app-plane.test.ts [hunk 1]
```diff
@@ -200,7 +200,53 @@ describe('CoreApplicationPlane', () => {
         ),
       },
     });
+    cdk.Aspects.of(app).add(new AwsSolutionsChecks());
```
Comment (suhussai) [inline test/core-app-plane.test.ts:203]: why are we adding this?

Comment (ujwalbukka) [reply test/core-app-plane.test.ts:203]: Removed it

**Comment by suhussai** [comment]
@ujwalbukka , is this still needed?

**Comment by ujwalbukka** [comment]
> @ujwalbukka , is this still needed?

We don't need this, closing it

**Pull request closed without merging**